cmd/cri-containerd/options/options.go

```diff
@@ -213,5 +225,8 @@ func defaultConfig() Config {
 		StatsCollectPeriod:  10,
 		SystemdCgroup:       false,
 		OOMScore:            -999,
+		EnableProfiling:     true,
+		ProfilingPort:       "6060",
```

Comment:
Where did you get this number? We should avoid port conflict with other system daemons.

We once have a production issue that kube-proxy has conflict port with node-problem-detector.

Reply:
I am sorry to say, I am not sure about  what  is better,  I use it for I have found cri-o use 6060 as port of pprof endpoint.  Maybe do not give default is better.  This problem is bothering me. Do you have any suggestions?

Reply:
Let's use 10009 or 10011, which is around our stream server port.

Reply:
@Random-Liu  Thanks a lot for your suggestion.  use 10011 as profiling port.

Reply:
The double 11 is coming.